examples/wrapper-types/src/wrappers.swift

```diff
@@ -0,0 +1,34 @@
+public struct FlatUser {
+    var userName: String
+    var accountId: Int32
+
+    public init(userName: String, accountId: Int32) {
+        self.userName = userName
+        self.accountId = accountId
+    }
+}
+
+extension FlatUser: Equatable, Hashable {
+    public static func == (lhs: FlatUser, rhs: FlatUser) -> Bool {
+        return lhs.userName == rhs.userName && lhs.accountId == rhs.accountId
+    }
+
+    public func hash(into hasher: inout Hasher) {
```

Comment:
I don't quite understand how this is relevant to the example? And even the implementation seems suspect - what does the same accountId but different name mean? :)

Reply:
I don't love this either, but it's needed because `WrappedTypesDemo` wants to implement `Hashable` by calling `hash()` on each of it's components.  I guess it's just one more reason why this is not a great example.